ws-messages/onConnect.json

```diff
@@ -1010,7 +1010,10 @@
                                 ],
                                 "requiresRestart": true,
                                 "title": "OTA index override file name",
-                                "type": "string"
+                                "type": [
```

Comment:
Hi! This file is only for local developer instance (mockups) and this change will not fix anything. You should make your change here https://github.com/Koenkk/zigbee2mqtt/blob/56589dccf156915dcc78e9c7ffd669d7e5597db0/lib/util/settings.schema.json#L347

Reply:
thanks)